src/Core/Domain/Product/ValueObject/RedirectType.php

```diff
@@ -35,16 +35,36 @@
  */
 class RedirectType
 {
+    /**
+     * Represents default configuration.
+     */
+    public const DEFAULT = 'default';
+
+    /**
+     * Represents value of no redirection. Product page with "Discontinued" message will be displayed, along with HTTP 200 response.
+     */
+    public const TYPE_SUCCESS_DISPLAYED = '200-displayed';
```

Comment:
Do we really need this type?
It sounds strange to return 200, but discontinued. The constant name also sounds strange for same reason :thinking:

Reply:
I have checked how the most successful ecommerce websites do this and many of them are returning normal 200 for discontinued products. 

I think some merchants want to preserve the positions in search engines, even after discontinuing the products. For example if you manufacture something and you still want people to easily find manuals and information about old products. If we offered 404 and 410 only, google would remove it from the results.